themes/ddbasic/sass/components/view/view.scss

```diff
@@ -66,11 +66,13 @@
     // Styleguide 2.1.9
 
     // Small grid
+    @at-root .grid-small#{&},
```

Comment:
@KristianB14: Thanks for explaining that. I am not well-versed in @at-root - nor in reading the documentation apparently. 

With that in mind the situation is a lot better.